modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/dynamic_include/customization_settings.jsp

```diff
@@ -106,7 +106,7 @@ boolean hasUpdateLayoutPermission = GetterUtil.getBoolean(request.getAttribute(C
 					"/layout_admin/reset_customization_view"
 				).buildString();
 
-				String resetCustomizationsViewURLString = "javascript:if (confirm('" + UnicodeLanguageUtil.get(resourceBundle, "are-you-sure-you-want-to-reset-your-customizations-to-default") + "')){submitForm(document.hrefFm, '" + HtmlUtil.escapeJS(resetCustomizationViewURL) + "');}";
+				String resetCustomizationsViewURLString = "javascript:Liferay.Util.openConfirmModal({message:\'" + UnicodeLanguageUtil.get(resourceBundle, "are-you-sure-you-want-to-reset-your-customizations-to-default") + "\',onConfirm: function(isConfirmed){if(isConfirmed){submitForm(document.hrefFm, \'" + HtmlUtil.escapeJS(resetCustomizationViewURL) + "\');}}})";
```

Comment:
Yes, I have done it, but not for all cases. It could take me a lot of time. I'm relying on CI as I already have done on Migration of JS use cases of confirm.